engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/runtime/PolyglotProxy.java

```diff
@@ -80,9 +83,9 @@ void leaveOrigin(InteropLibrary node, Object prev) {
   public boolean isNull(
       @CachedLibrary("this.delegate") InteropLibrary nulls,
       @CachedLibrary("this") InteropLibrary node,
-      @CachedLibrary(limit = "5") InteropLibrary errors,
-      @Cached @Cached.Exclusive ContextRewrapExceptionNode contextRewrapExceptionNode,
-      @Cached @Cached.Exclusive BranchProfile profile) {
+      @Shared("interop") @CachedLibrary(limit = "5") InteropLibrary errors,
```

Comment:
Please note there is `InteropLibrary` for `nulls` and another one for `errors` - e.g. they are intentionally separated. Are benchmarks OK even after all this `@Shared` annotation adding?